src/components/cli.ts

```diff
@@ -15,7 +15,7 @@ limitations under the License.
 */
 
 import * as fs from "fs";
-import * as path from "path";
+import path from "path";
```

Comment:
`path` gets imported like this, but `fs` above does not?

Reply:
apparently we needed the default path but not for fs